src/Config/SDK/ComponentProvider/Logs/LogRecordExporterOtlpHttp.php

```diff
@@ -0,0 +1,84 @@
+/**
+ * @implements ComponentProvider<LogRecordExporterInterface>
+ */
+#[PackageDependency('open-telemetry/exporter-otlp', '^1.0.5')]
+final class LogRecordExporterOtlpHttp implements ComponentProvider
+{
+    /**
+     * @param array{
+     *     endpoint: string,
+     *     certificate: ?string,
+     *     client_key: ?string,
+     *     client_certificate: ?string,
+     *     headers: list<array{name: string, value: string}>,
+     *     headers_list: ?string,
+     *     compression: 'gzip'|null,
+     *     timeout: int<0, max>,
+     *     encoding: 'protobuf'|'json',
+     * } $properties
+     */
+    public function createPlugin(array $properties, Context $context): LogRecordExporterInterface
+    {
+        $headers = array_column($properties['headers'], 'value', 'name') + MapParser::parse($properties['headers_list']);
+
+        return new LogsExporter(Registry::transportFactory('http')->create(
+            endpoint: $properties['endpoint'],
+            contentType: match ($properties['encoding']) {
+                'protobuf' => ContentTypes::PROTOBUF,
+                'json' => ContentTypes::JSON,
+            },
+            headers: $headers,
+            compression: $properties['compression'],
+            timeout: $properties['timeout'],
+            cacert: $properties['certificate'],
+            cert: $properties['client_certificate'],
+            key: $properties['client_certificate'],
+        ));
+    }
+
+    public function getConfig(ComponentProviderRegistry $registry, NodeBuilder $builder): ArrayNodeDefinition
+    {
+        $node = $builder->arrayNode('otlp_http');
+        $node
+            ->children()
+                ->enumNode('encoding')->defaultValue('protobuf')->values(['protobuf', 'json'])->end()
+                ->scalarNode('endpoint')->defaultValue('http://localhost:4318/v1/logs')->validate()->always(Validation::ensureString())->end()->end()
+                ->scalarNode('certificate')->defaultNull()->validate()->always(Validation::ensureString())->end()->end()
+                ->scalarNode('client_key')->defaultNull()->validate()->always(Validation::ensureString())->end()->end()
+                ->scalarNode('client_certificate')->defaultNull()->validate()->always(Validation::ensureString())->end()->end()
+                ->arrayNode('headers')
+                    ->arrayPrototype()
+                        ->children()
+                            ->scalarNode('name')->isRequired()->cannotBeEmpty()->end()
+                            ->scalarNode('value')->defaultNull()->validate()->always(Validation::ensureString())->end()->end()
+                        ->end()
+                    ->end()
+                ->end()
+                ->scalarNode('headers_list')->defaultNull()->validate()->always(Validation::ensureString())->end()->end()
+                ->enumNode('compression')->values(['gzip'])->defaultNull()->end()
+                ->integerNode('timeout')->min(0)->defaultValue(10)->end()
```

Comment:
```suggestion
                ->integerNode('timeout')->min(0)->defaultValue(10000)->end()
```

Unrelated to this PR, but `timeout` is specified in milliseconds, must be changed for all OTLP exporters.